common/text.h

```diff
@@ -6,3 +6,5 @@ void textInit(void);
 int textGetLang(void);
 const char* textGetString(StrId id);
 u64 textGetLanguageCode(void);
+uint32_t getXCoordinate(u32 font,uint32_t rX, const char* text,const char align);
```

Comment:
Move to common.h/font.c, similar code is already located there.

Reply:
done